src/css.js

```diff
@@ -113,6 +113,15 @@ function getWidthOrHeight( elem, name, extra ) {
 		styles = getStyles( elem ),
 		isBorderBox = jQuery.css( elem, "boxSizing", false, styles ) === "border-box";
 
+	// Support: IE11+
+	// Fix for edge case in IE 11. See https://github.com/jquery/jquery/issues/1764
+	if (!support.reliableDimensionsInFullscreenMode && window.top !== window.self &&
```

Comment:
We put spaces after the opening paren and before the closing one, same below when using `Math.round`. For more details, see https://contribute.jquery.org/style-guide/js/#spacing